src/ansys/templates/python/solution/cookiecutter.json

```diff
@@ -10,8 +10,10 @@
   "__short_description": "",
   "__pkg_name": "ansys-solutions-{{ cookiecutter.solution_name.lower().replace('_', '-') }}",
   "__pkg_namespace": "ansys.solutions.{{ cookiecutter.solution_name.lower().replace('-', '_') }}",
-  "__requires_python": "3.8",
+  "__requires_python": "3.9",
   "__repository_url": "",
+  "__documentation_url": "",
+  "build_system": "poetry",
   "__build_system": "poetry",
```

Comment:
@iazehaf I removed the `build_system` line, but we have to keep `__build_system` because it's used in getting_started/index.rst: https://github.com/ansys/ansys-templates/actions/runs/9036325176/job/24832984544#step:5:387

Reply:
Ok. Thanks.